packages/stats/gbstats/bayesian/tests.py

```diff
@@ -313,3 +323,141 @@ def compute_result(self) -> BayesianTestResult:
                 result, self.traffic_proportion_b, self.phase_length_days
             )
         return result
+
+
+class GaussianEffectABTest(BayesianABTest):
+    def __init__(
+        self,
+        stat_a: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        stat_b: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        config: EffectBayesianConfig,
+    ):
+        super().__init__(stat_a, stat_b, config)
+        self.prior_effect = config.prior_effect
+        self.stat_a = stat_a
+        self.stat_b = stat_b
+
+    def compute_result(self):
+        if self.stat_a.mean == 0 and self.relative:
+            return self._default_output(BASELINE_VARIATION_ZERO_MESSAGE)
+        if self.has_empty_input():
+            return self._default_output(NO_UNITS_IN_VARIATION_MESSAGE)
+        if self._has_zero_variance():
+            return self._default_output(ZERO_NEGATIVE_VARIANCE_MESSAGE)
+
+        self.compute_frequentist_moments()
+        self.construct_prior_by_variation()
+
+        mu_a, sd_a = Norm.posterior(
+            [
+                self.prior_a.mean,
+                np.sqrt(self.prior_a.variance),
+                self.prior_a.pseudo_n,
+            ],
+            [
+                self.stat_a.mean,
+                self.stat_a.stddev,
+                self.stat_a.n,
+            ],
+        )
+        mu_b, sd_b = Norm.posterior(
+            [
+                self.prior_b.mean,
+                np.sqrt(self.prior_b.variance),
+                self.prior_b.pseudo_n,
+            ],
+            [
+                self.stat_b.mean,
+                self.stat_b.stddev,
+                self.stat_b.n,
+            ],
+        )
+        self.mean_diff = mu_b - mu_a
+        self.std_diff = np.sqrt(sd_a**2 + sd_b**2)
+        risk = [self.risk]
+        self.var_diff = frequentist_variance(
+            sd_a**2, mu_a, 1, sd_b**2, mu_b, 1, self.relative
```

Comment:
yes, because `var_a` in `frequentist_variance` refers to the distributional variance, and divides by `n_a` to get the variance of the mean.  `Norm.posterior` outputs the variance for the mean, so we don't want to divide it again.